src/components/MyAccount/Settings/AccountSettingsDisplayOptions.test.tsx

```diff
@@ -27,7 +27,7 @@ describe("AccountSettingsDisplayOptions", () => {
     beforeEach(() => {
       render(
         <AccountSettingsDisplay
-          patron={{ ...processedPatron, notificationPreference: "Mobile" }}
+          patron={{ ...processedPatron, notificationPreference: "p" }}
```

Comment:
I think it would help if `notificationPreference` in the `Patron` interface is defined as the set of values `"z" | "p"` so it's not confusing whether the value is the entire word or the character value.